lib/parser.dart

```diff
@@ -1816,7 +1818,7 @@ class InBodyPhase extends Phase {
     if (tree.formPointer != null) {
       return;
     }
-    var formAttrs = <dynamic, String>{};
+    var formAttrs = LinkedHashMap<dynamic, String>();
```

Comment:
Why this change?

Reply:
It needs to be typed as `LinkedHashMap` below.

https://github.com/dart-lang/html/blob/25806e5e8e30f68aee4f108f229a22d9706ed589/lib/src/token.dart#L25